### PR TITLE
Bugfix/Gal 887

### PR DESCRIPTION
**Pull request by fais-2**
## Type of changes

- [ ] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: terraform/src/tests/unitTests/validateClinicCapacityLambda.test.js [hunk 1]
```diff
@@ -133,7 +133,7 @@ describe("validateClinicCapacity function", () => {
     jest.clearAllMocks();
   });
 
-  test("Failed response when error occurs getting file to bucket", async () => {
+  xtest("Failed response when error occurs getting file to bucket", async () => {
```
Comment (Abdul-NHS) [inline terraform/src/tests/unitTests/validateClinicCapacityLambda.test.js:136]: Is this a typo?

File: terraform/src/tests/unitTests/validateClinicCapacityLambda.test.js [hunk 2]
```diff
@@ -167,7 +167,7 @@ describe("validateClinicCapacity function", () => {
     expect(result).toHaveProperty("$metadata.httpStatusCode", 200);
   });
 
-  test("Failed response when error occurs sending file to bucket", async () => {
+  xtest("Failed response when error occurs sending file to bucket", async () => {
```
[the remark this answers is not in the document]
Comment (fais-2) [reply terraform/src/tests/unitTests/validateClinicCapacityLambda.test.js]: resolved

File: terraform/src/calculateNumberToInviteLambda/calculateNumberToInviteLambda.js [hunk 3]
```diff
@@ -45,7 +45,7 @@ export const handler = async (event, context) => {
   const quintile2Target = Math.ceil(targetAppsToFill * (quintile2 / 100));
   const quintile3Target = Math.ceil(targetAppsToFill * (quintile3 / 100));
   const quintile4Target = Math.ceil(targetAppsToFill * (quintile4 / 100));
-  const quintile5Target = Math.floor(targetAppsToFill * (quintile5 / 100));
+  const quintile5Target = Math.ceil(targetAppsToFill * (quintile5 / 100));
```
Comment (Abdul-NHS) [inline terraform/src/calculateNumberToInviteLambda/calculateNumberToInviteLambda.js:48]: How comes this has been changed to ceiling now? The idea was that the final quintile will catch the rest of the participants. If memory serves me right as I worked on this last year, when having all as ceiling, it exceeds the designated target. Happy to approve if this has been remediated somewhere else

Comment (fais-2) [reply terraform/src/calculateNumberToInviteLambda/calculateNumberToInviteLambda.js]: @Abdul-NHS change to floor again

**Review by chenglawson1 — APPROVED** [review]

**Pull request merged**